.env.test

```diff
@@ -5,6 +5,11 @@ DATABASE_PASSWORD=root
 DATABASE_HOST=localhost
 DATABASE_PORT=5430
 DATABASE_URL=postgresql://postgres:root@db/postgres
+
 SECRET_KEY=test
+
 GF_SECURITY_ADMIN_PASSWORD=admin
-GRAFANA_HOST=grafana-dev
\ No newline at end of file
+GRAFANA_HOST=grafana-dev
+
+CELERY_BROKER_URL=redis://localhost:6379/0
+CELERY_RESULT_BACKEND=redis://localhost:6379/0
```

Comment:
Yes 👍 If we run tests, the process with the test runs outside of the docker container. That's why we have to expose the port of the Redis database and use the localhost. The development and production container run inside docker and use the docker-dns names. Lucas and I considered running the tests inside docker but decided against it because it's not an infrastructure.